deploy/crds/operator.ibm.com_v1alpha1_authentication_cr.yaml

```diff
@@ -96,6 +96,10 @@ spec:
     openshiftPort: 443
     roksEnabled: true
     nonceEnabled: true
+    claimsSupported: "name,family_name,display_name,given_name,preferred_username"
+    claimsMap: "name=\"givenName\" family_name=\"givenName\" given_name=\"givenName\" preferred_username=\"givenName\"
+    display_name=\"displayName\""
+    scopeClaim: "profile=\"name,family_name,display_name,given_name,preferred_username\""
     bootstrapUserId: ""
```

Comment:
bootstraUserId should be on like 51 if it's a required field

Reply:
bootStrap is what i expect the user to set, for the other three there are defaults. But should be editable if customer wants.